### PR TITLE
Chapter 7

### PR DESCRIPTION
**Pull request by aisharina99**
ready

### CODE REVIEW

**Comment by aisharina99** [comment]
ready

**Comment by ngocvt-0484** [comment]
please combine 4 commits to 1 commit

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss]: delete this file because not use

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ngocvt-0484) [inline app/helpers/static_pages_helper.rb]: same
delete it please

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
+  get 'users/show'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove this line

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,9 @@
...
+  get 'users/new'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove this line

File: test/controllers/users_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get users_show_url
+    assert_response :success
+  end
```
Comment (ngocvt-0484) [inline test/controllers/users_controller_test.rb]: dont need do test, so please delete all file test on  test folder

Comment (aisharina99) [reply test/controllers/users_controller_test.rb]: by all file did you mean, (application_system_test_case.rb) and (test_helper.rb) and (connection_test.rb) too?

Comment (aisharina99) [reply test/controllers/users_controller_test.rb:7]: Screenshot from 2021-09-14 16-18-32
I want to ask what should I do if rails server become like this?

Comment (aisharina99) [reply test/controllers/users_controller_test.rb]: I want to ask, I rebased the 4 commits and then there are conflicting files. When I resolve the conflicts there are 2 commit and I cant rebase it become fatal error.

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Review by ghost — COMMENTED** [review]
You have 2 commit, please combine into 1

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Comment by aisharina99** [comment]
ready

**Pull request merged**